index.html

```diff
@@ -233,11 +237,159 @@ <h3 class="timeline__title">18th Mar 5:00 PM :- Prize Distribution & Feedback Se
             For HINT 3.0, we will be considering all teams equally on merit - as we always have. In order to bridge this troublesome gap, we assure you a reservation of 20% seats for girls leaving no excuse to prove a point Beyoncé made forever ago- who run the world? GIRLS!
             
-        </div></div>
+        </div></div>*
         </div>
     </section>
 </section>
+<section id="awards" class="b-item">
+    <div class="container-fluid">
+    <center><h1>Special Prizes</h1></center>
+    <div class="row">
+    
+        <div class="card">
+                <div class="logo">
+                    <img src="static/css/tech.png" style="height : 110px; width : 160px; margin : 23px;  ">
+                </div>
+                <div class="descp">
+                        <h3>INR 15,000</h3>
+                        Amazon Gift-card for the Best hack/website submitted on a .tech domain
+
+                    </div>
+            </div>
+            <div class="card">
+                    <div class="logo">
+                        <img src="static/css/zulip.png">
 
+                        <div class="row" style="font-weight : bold; margin-top : -20px; font-size : 20px;">Zulip</div>
+                    </div>
+                    <div class="descp">
+                            <h3>100$</h3>
+                            For API Winner
+                       
+                    </div>
+                </div>
+            <div class="card">
+                    <div class="logo">
+                        <img src="static/css/jet.png" style="margin : 15px; ">
+                    </div>
+                    <div class="descp">
+                            <h3 style=" margin-top : 16px; font-weight : bold;">Lifetime premium</h3>
+                            Lifetime premium license for JetBrains IDEs to JetBrains Quiz winners
+
+                        </div>
+                </div>
+                <div class="card">
+                        <div class="logo">
+                            <img src="static/css/sketch.png" style="height : 110px; margin : 23px;  ">
+                            <div class="row" style="font-weight : bold; margin-top : -50px; font-size : 20px;">Sketch</div>
+                        </div>
+                        <div class="descp">
+                               <h3>99$</h3>
+                               $99 credits for top 3 teams
+                            </div>
+                    </div>
+            <div class="card">
+                    <div class="logo">
+                        <img src="static/css/wolfram.png" style="height :180px; width : 270px;">
+                    </div>
+                    <div class="descp">
+                            <h3>5 lakhs </h3>
+                            Premium API worth 5 lakhs to top 15 teams
```

Comment:
make this `The Wolfram Award for everyone on your top fifteen teams,with value of each individual award is $375.00`